playwright_example.env

```diff
@@ -1,15 +1,19 @@
-# SKUs for QE user contentPlaywrightUserAdmin: MCT4022,MCT3718,MCT3695,ES0113909
-
+#SKUs for the admin user:
+# MCT4022,MCT3718,MCT3695,ES0113909
+#
 # ------------- LOGINS -------------
-# - ADMIN / DEFAULT
-USER1USERNAME="contentPlaywrightUserAdmin" # Required
-USER1PASSWORD=""                           # Required
-ORG_ID_1="1234"                            # org id to register for registration tests
-ACTIVATION_KEY_1="MyKey"                   # activation Key used for testing
+# Admin user
+
+ADMIN_USERNAME="" # Required
+ADMIN_PASSWORD="" # Required
+
+ORG_ID_1="1234"                            # org id for registration tests
+ACTIVATION_KEY_1="<activation_key>"        # activation key used for testing
+
+# read-only user (same ORG as the admin)
 
-# - READ-ONLY (same ORG as the admin)
```

Comment:
_style nit: the all caps comments seam to be nicer for distinction between parts of the env file 💅🏼 
(don't bother fixing unless you really want to 😄)_